tutorials/katacoda/getting-started/setup.md

```diff
@@ -0,0 +1,56 @@
+## Before you begin
+
+Start an experimental Kubernetes cluster in Katacoda with command
+`launch.sh`{{execute}}.
+
+Run `kubectl cluster-info`{{execute}} in the terminal to check its status.
+You should see the following outputs:
+
+```
+Kubernetes master is running at ...
+KubeDNS is running at ...
+
+...
+```
+
+## Installing Tekton
+
+To add Tekton to this experimental Kubernetes cluster, execute the command
+below:
+
+`kubectl apply --filename https://storage.googleapis.com/tekton-releases/latest/release.yaml`{{execute}}
+
+It may take a few moments for the installation to complete. To monitor the
+progress, run the following command:
+
+`kubectl get pods --namespace tekton-pipelines`{{execute}}
+
+Every component listed in the output should have the status `running`.
+
+## Getting the code
+
+We’ve put everything you need for this scenario in a GitHub repository.
+To clone this repository, run the following command:
+
+`git clone https://github.com/tektoncd/website`{{execute}}
+
+Open the directory `tekton-example/getting-started/src` with the file explorer.
```

Comment:
```suggestion
Open the directory `website/tutorials/getting-started/src` with the file explorer.
```

There are a few other references to the path in other files that also need to be updated.